scripts/build.sh

```diff
@@ -24,6 +24,14 @@ CI_BUILD=false
 VERBOSE=false
 PROJECT_NAME_PATTERNS=
 
+#
```

Comment:
Need to update test.sh?

Reply:
Not required.

Reply:
I think dotnet cli(dotnet/<any_repo>) runs tests with `build.cmd`,  I think `source-build` expects tests should run as part of `build`.

Reply:
> I think source-build expects tests should run as part of build.

Didn't see any API related to running tests in the repo api docs: https://github.com/dotnet/source-build/blob/dev/release/2.0/Documentation/RepoApi.md. It only mentions `build.cmd/sh`.